res/GAMEHELP/WOLFEN84

```diff
@@ -0,0 +1,24 @@
+       Find the battle plans and 
```

Comment:
game help is coming in https://github.com/a2-4am/4cade/pull/165 so can you remove this file